tests/general/test_torch_backend.py

```diff
@@ -1,16 +1,16 @@
 import torch
 import pytest
-from kymatio.backend.torch_backend import ModulusStable, modulus
+from kymatio.backend.torch_backend import ModulusStable, TorchBackend
 
 
 def test_modulus(random_state=42):
     """
     Tests the stability and differentiability of modulus
     """
-
+    backend = TorchBackend()
```

Comment:
Why not just import `backend` from `kymatio.backend.torch_backend`? It's already been instantiated and that's what the main code will use.

Reply:
especially now, since it's actually not instantiated

Reply:
though that said, it might be more modular to do it like above, just without the parentheses